include/moveit_visual_tools/moveit_visual_tools.h

```diff
@@ -323,33 +323,40 @@ class MoveItVisualTools : public rviz_visual_tools::RvizVisualTools
                              const rviz_visual_tools::colors& color = rviz_visual_tools::GREEN);
 
   /**
-   * \brief Create a MoveIt Collision block at the given pose
+   * \brief Create/Publish a MoveIt Collision block at the given pose
    * \param pose - location of center of block
    * \param name - semantic name of MoveIt collision object
    * \param size - height=width=depth=size
    * \param color to display the collision object with
    * \return true on sucess
    **/
+  moveit_msgs::CollisionObject createCollisionBlock(const geometry_msgs::Pose& block_pose, const std::string& name,
+                                                    double block_size);
```

Comment:
is it possible for the CollisionObject msgs type to support color as well?

Reply:
We would have to change the msg

Reply:
we would have to change the msg